ports/renesas-ra/boards/EK_RA4M1/ra_cfg/fsp_cfg/r_sci_uart_cfg.h

```diff
@@ -0,0 +1,16 @@
+#ifndef R_SCI_UART_CFG_H_
```

Comment:
These config files look exactly the same for all boards.  In the interest of trying to reduce the number of files, and to make it easier to add new boards, can this config file be removed from each board and added either in `ports/renesas-ra`, or `ports/renesas-ra/ra`, or maybe in a new subdirectory called `ports/renesas-ra/fsp_cfg`?

Reply:
See #11846 for my proposal to simplify this configuration.